src/gcp_scanner/crawl.py

```diff
@@ -563,8 +537,7 @@ def get_bq(project_id: str,
 
       for dataset in response.get("datasets", []):
         dataset_id = dataset["datasetReference"]["datasetId"]
-        bq_datasets[dataset_id] = get_bq_tables(project_id, dataset_id, service)
-
+        bq_datasets[dataset_id]=get_bq_tables(project_id,dataset_id, service)
```

Comment:
Keep the whitespace

Reply:
added the white-space

Reply:
@RahulDubey391 Are you sure this is resolved?